HACKING.rst

```diff
@@ -7,14 +7,14 @@ Next, each time you start working, enter the environment::
 
   source .venv/bin/activate
 
-The HaaS has an unspecified dependency on the ``moc-rest`` library:
-
-.. https://github.com/cci-moc/moc-rest
+The HaaS has an unspecified dependency on the ``moc-rest`` library, which we
+hope to `remove in the future <https://github.com/CCI-MOC/haas/issues/394>`_.
+It may be found at: https://github.com/cci-moc/moc-rest .
 
 The recommended way for developers to satisfy this is to clone that repository,
-and then (while in the virtual environment)::
+and then while in the virtual environment::
 
-  pip install -e ${path_to_moc_rest_clone}
+  pip install 'git+https://github.com/CCI-MOC/moc-rest'
```

Comment:
It looks to me like you no longer need to clone the repository manually, and that pip will do it for you.  Also, is the -e flag no longer needed, when using this notation?